### PR TITLE
refactor loading of comments

### PR DESCRIPTION
**Pull request by jaybe78**
Currently we fetch all comments for all threads displayed in the UI.
From the performance point of view, it's not ideal.
We can improve that by fetching only the number of unique users who commented on the threads and 2 user avatars for each thread, since we only show 2 on each row.

## Description

1) I've removed the queries which fetch `comments` in `/bulkOffChain` and `/bulkThreads`.
2) I created a new endpoint `/threadsUserCountAndAvatar` which count the number of addresses for each thread and fetch among that the first 2 addresses.
The endpoint returns `{ rootId: string, addresses: [], count: number }`
3) I've also modified the query which loads threads in `/bulkThreads` with an extra field `latest_comm_created_at` so that I can sort threads by latest comment
4) prettier on each file modified

## Have proper tags been added (for bug, enhancement, breaking change)?
- [ ] yes

## Does this PR affect any server routes?
- [ ] yes, and they are tested: [enter the % coverage here]
- [ ] yes, and they are not tested: [enter the % coverage here]
- [ ] yes, but I did not run tests
- [ ] no

### CODE REVIEW

**Comment by jnaviask** [comment]
To be clear, this is still WIP?

**Comment by jaybe78** [comment]
> To be clear, this is still WIP?

yes it is

File: client/scripts/controllers/server/threads.ts [hunk 1]
```diff
@@ -112,6 +125,9 @@ export const modelFromServer = (thread) => {
     offchainVotingOptions: offchain_voting_options,
     offchainVotingEndsAt: offchain_voting_ends_at,
     offchainVotingNumVotes: offchain_voting_votes,
+    latest_comm_created_at: latest_comm_created_at
```
Comment (gdjohnson) [inline client/scripts/controllers/server/threads.ts:128]: lil nitpicky but can we use camelCase for all our client-side variables?

File: server/routes/bulkThreads.ts [hunk 2]
```diff
@@ -138,11 +142,12 @@ const bulkThreads = async (models: DB, req: Request, res: Response, next: NextFu
         offchain_voting_options: t.offchain_voting_options,
         offchain_voting_votes: t.offchain_voting_votes,
         offchain_voting_ends_at: t.offchain_voting_ends_at,
+        latestCommCreatedAt: latest_comm_created_at,
```
Comment (zakhap) [inline server/routes/bulkThreads.ts:145]: this option should be standardized with the others or the others to it.

**Pull request merged**